modules/installation-vsphere-infrastructure.adoc

```diff
@@ -26,11 +26,7 @@ You must install the {product-title} cluster on a VMware vSphere version 6 or 7
 
 |Storage with in-tree drivers
 |vSphere 6.5 and later
-|This plug-in creates vSphere storage by using the in-tree storage drivers for vSphere included in {product-title} and can be used when vSphere CSI drivers are not available.
-
-|Storage with vSphere CSI driver
-|vSphere 6.7U3 and later
-|This plug-in creates vSphere storage by using the standard Container Storage Interface. The vSphere CSI driver is provided and supported by VMware.
```

Comment:
Per https://bugzilla.redhat.com/show_bug.cgi?id=1928861 - removing the CSI driver statements in current 4.x docs to avoid confusion about supportability of that driver in OCP.
@sferich888 or @reestr PTAL to ack this change